package.json

```diff
@@ -83,14 +83,16 @@
     "eslint": "^7.4.0",
     "faker": "^4.1.0",
     "grunt-bower-task": "^0.5.0",
-    "husky": "^4.3.0",
+    "husky": "^5.0.0-beta.0",
+    "lint-staged": "^10.4.2",
     "mocha": "^8.1.0",
     "mocha-junit-reporter": "^2.0.0",
     "mocha-lcov-reporter": "^1.3.0",
     "mock-req-res": "^1.2.0",
     "mockdate": "^3.0.2",
     "nodemon": "^2.0.3",
     "nyc": "^15.0.1",
+    "prettier": "^2.1.2",
```

Comment:
why was prettier added?

Reply:
```suggestion
```

Reply:
This as well, I was experimenting this before committing. Will get this rectified. :) Thanks for the patience